package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "@rolemodel/rolemodel-design-system",
-  "version": "0.1.3-beta",
+  "name": "@rolemodel/optics",
+  "version": "0.2.0",
   "description": "The design system RoleModel uses to kickstart projects with a consistent starting point",
```

Comment:
```suggestion
  "description": "Optics is an scss package that provides base styles and components that can be integrated and customized in a variety of projects.",
```

Reply:
@Jeremy-Walton Thinking this should be the same as what is stated within the `README.md`.

Reply:
Good call. Updated